pom.xml

```diff
@@ -22,6 +22,7 @@
           Note that the `dockerhost.ip` property is computed from the IP address of DOCKER_HOST, which will
           work on all platforms. We'll set some of these as system properties during integration testing.
         -->
+        <version.vitess>12.0.0</version.vitess>
```

Comment:
Could you instead please update the version in the [Debezium BOM](https://github.com/debezium/debezium/blob/main/debezium-bom/pom.xml#L26-L31)? All the dependency versions are coordinated via that one.

Reply:
Thanks for the pointer! Since the flag is not available in Vitess 9.0.0 release, we need this PR first https://github.com/debezium/debezium/pull/2951

Reply:
Merged that PR. Note if both PRs would have used the same branch name, this PR here would have been built against that update over there, without having to merge that other PR first. That's a nice way to avoid merging the upstream PR first and then noticing it caused issues and needs backing out. In this case it was not a problem, as we had seen that the update to 12 works here already. So just mentioning it as some context info :)

Reply:
Ah, didn't know that. TIL

Do you know how do I bump the reference in this branch to pick up the version change? I removed this line and tests are failing.

Reply:
Or is it OK if I remove this two lines in pom.xml and we merge as is?

Reply:
I hope it will pass actually. This build fetches the main branch from the core repo (with your version update) and builds against that (in absence of a branch with the same name over there).

Reply:
Ah yeah, you are right! I guess it's just my local branch issue